### PR TITLE
[ADD] Expenses: approve expense reports

### PR DESCRIPTION
**Pull request by larm-odoo**
Original Expenses doc too long, breaking off the approving expense reports section into its own doc, and making necessary updates to version 17 (original was written for 16).

[Project Task Card ](https://www.odoo.com/odoo/project/3835/tasks/4076086?cids=3)for this PR.

### CODE REVIEW

**Comment by larm-odoo** [comment]
Hi @ksc-odoo  - you are my first-round reviewer for this doc!

Please note, the changes you requested form the original expenses.rst are NOT on this PR. I will have to forward port the changes when I fix the merge conflict that will arise once the other PR's are published =). I know this will occur, so I wanted to explain why you are going to see that original expense.rst in the 'old' format without your proposed changes form my other PR.

File: content/applications/finance/expenses/approve_expenses.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+.. image:: approve-expenses/expense-reports-list.png
+   :align: center
+   :alt: Reports to validate are found on the Reports to Approve page.
+
+Approve expense reports
+=======================
+
+Expense reports can be approved in two ways: :ref:`individually <expenses/individual>` or
+:ref:`several at once <expenses/multiple>`.
```
Comment (ksc-odoo) [inline content/applications/finance/expenses/approve_expenses.rst:32]:
```suggestion
:ref:`in bulk <expenses/multiple>`.
```

File: content/applications/finance/expenses/approve_expenses.rst [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+.. important::
+   Only reports with a status of :guilabel:`Submitted` can be approved. It is recommended to display
+   only these reports by enabling the :guilabel:`Submitted` filter in the left side column.
```
Comment (ksc-odoo) [inline content/applications/finance/expenses/approve_expenses.rst:36]:
```suggestion
   Only reports with a status of :guilabel:`Submitted` can be approved.
   
   It is recommended to only display these reports by ticking the checkbox beside the
   :guilabel:`Submitted` filter, in the left column, under the :guilabel:`Status` section.
```

File: content/applications/finance/expenses/approve_expenses.rst [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+   If a report is **not** able to be approved, the :guilabel:`Approve Report` button **does not**
+   appear.
+
+.. _expenses/individual:
+
+Approve individual reports
+--------------------------
+
+To approve an individual report, navigate to :menuselection:`Expenses app --> Expense Reports`, and
+click on an individual report to view the details.
```
Comment (ksc-odoo) [inline content/applications/finance/expenses/approve_expenses.rst:47]:
```suggestion
click on an individual report to view the report form.
```

File: content/applications/finance/expenses/approve_expenses.rst [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+In this view, several options are presented: :guilabel:`Approve`,  :guilabel:`Refuse`, or
```
Comment (ksc-odoo) [inline content/applications/finance/expenses/approve_expenses.rst:49]:
```suggestion
From here, several options are presented: :guilabel:`Approve`,  :guilabel:`Refuse`, and
```

File: content/applications/finance/expenses/approve_expenses.rst [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+.. _expenses/multiple:
+
+Approve multiple reports
+------------------------
+
+To approve multiple expense reports at once, first navigate to :menuselection:`Expenses app -->
+Expense Reports` to view a list of expense reports. Next, select the reports to approve by ticking
+the checkbox next to each report being approved, or tick the checkbox next to :guilabel:`Employee`
+to quickly select **all** the reports in the list.
```
Comment (ksc-odoo) [inline content/applications/finance/expenses/approve_expenses.rst:62]:
```suggestion
the checkbox next to each report being approved, or tick the checkbox next to the
:guilabel:`Employee` column title to select all the reports in the list.
```

File: content/applications/finance/expenses/approve_expenses.rst [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+
+Expense reports can be approved in two ways: :ref:`individually <expenses/individual>` or
+:ref:`several at once <expenses/multiple>`.
+
+.. important::
+   Only reports with a status of :guilabel:`Submitted` can be approved. It is recommended to display
+   only these reports by enabling the :guilabel:`Submitted` filter in the left side column.
+
+   If a report is **not** able to be approved, the :guilabel:`Approve Report` button **does not**
+   appear.
```
Comment (ksc-odoo) [inline content/applications/finance/expenses/approve_expenses.rst:39]:
```suggestion
   appear on the :guilabel:`All Reports` page.
```

File: content/applications/finance/expenses/approve_expenses.rst [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+
+.. image:: approve-expenses/approve-report.png
+   :align: center
+   :alt: Approve multiple reports by clicking the checkboxes next to each report.
+
+.. tip::
+   Team managers can easily view all the expense reports for just their team members. While in the
+   :guilabel:`Expense Reports` view, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` in the
+   right-side of the search box, then click :guilabel:`My Team` in the :icon:`fa-filter`
+   :guilabel:`Filters` section.
```
Comment (ksc-odoo) [inline content/applications/finance/expenses/approve_expenses.rst:74]:
```suggestion
   It is possible for team managers to view all the expense reports for just their team members.
   
   To do so, while on the :guilabel:`All Reports` page, click the :icon:`fa-caret-down` :guilabel:`(down arrow)`
   to the right of the search bar, then click :guilabel:`My Team` in the :icon:`fa-filter` :guilabel:`Filters` section.
```

File: content/applications/finance/expenses/approve_expenses.rst [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+      :align: center
+      :alt: Select the My Team filter.
+
+Refuse expense reports
+======================
+
+Expense reports can **only** be refused on the individual expense report, and **not** from the
+:guilabel:`All Reports` dashboard. To open an individual expense report, navigate to
+:menuselection:`Expenses app --> Expense Reports`, then click on an individual expense report to
+view the report details.
```
Comment (ksc-odoo) [inline content/applications/finance/expenses/approve_expenses.rst:88]:
```suggestion
view the report form.
```

File: content/applications/finance/expenses/approve_expenses.rst [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+
+If more information is needed, such as a missing receipt, communicate any necessary information
+requests from the chatter. On the individual expense report, click :guilabel:`Send message` to open
```
Comment (ksc-odoo) [inline content/applications/finance/expenses/approve_expenses.rst:91]:
```suggestion
requests in the *chatter* of the report form. On the individual expense report, click :guilabel:`Send message` to open
```

File: content/applications/finance/expenses/approve_expenses.rst [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+a message text box.
+
+Type in a message, tagging the proper people, and post it to the chatter by clicking
```
Comment (ksc-odoo) [inline content/applications/finance/expenses/approve_expenses.rst:94]:
```suggestion
Type in a message, tagging the proper people, and post it to the *chatter* by clicking
```

File: content/applications/finance/expenses/approve_expenses.rst [hunk 11]
```diff
@@ -0,0 +1,115 @@
...
+:guilabel:`Send`. The message is posted in the chatter, and the tagged people are notified via
```
Comment (ksc-odoo) [inline content/applications/finance/expenses/approve_expenses.rst:95]:
```suggestion
:guilabel:`Send`. The message is posted in the chatter, and the tagged people are notified, via
```

File: content/applications/finance/expenses/approve_expenses.rst [hunk 12]
```diff
@@ -0,0 +1,115 @@
...
+email.
+
+.. note::
+   The only people that can be tagged in a message are *followers*. To see who is a follower, click
```
Comment (ksc-odoo) [inline content/applications/finance/expenses/approve_expenses.rst:99]:
```suggestion
   The only people that can be tagged in a message are *followers* of the specific report. To see who is a follower, click
```

File: content/applications/finance/expenses/approve_expenses.rst [hunk 13]
```diff
@@ -0,0 +1,115 @@
...
+
+To refuse an expense report, click :guilabel:`Refuse`, and a :guilabel:`Refuse Expense` pop-up
+window appears. Enter a brief explanation for the refusal beneath the :guilabel:`REASON TO REFUSE
+EXPENSE` field, then click :guilabel:`Refuse`.
+
+.. image:: approve-expenses/refuse-expense.png
+   :align: center
+   :alt: Send messages in the chatter.
+
+One the expense report is refused, the status changes to :guilabel:`Refused`, and the only button
```
Comment (ksc-odoo) [inline content/applications/finance/expenses/approve_expenses.rst:114]:
```suggestion
Once the expense report is refused, the status changes to :guilabel:`Refused`, and the only button
```

**Review by ksc-odoo — APPROVED** [review]
Hey @larm-odoo -- just finished my Peer Review of this PR. Great job. Once you address the comments/feedback I left for you, feel free to move this to the next stage. Thanks! 👍

**Comment by larm-odoo** [comment]
Hi @StraubCreative - KSC did the first round, so that means you get to do the final review  =)

**Review by StraubCreative — APPROVED** [review]
Solid doc @larm-odoo 👍 
On to you, @samueljlieber!

**Comment by larm-odoo** [comment]
Finally, I got all the conflicts fixed, @samueljlieber! It's all ready for you =)

**Review by samueljlieber — APPROVED** [review]
Looks good to me @larm-odoo! Nice work on this PR :)
.....
@robodoo r+

**Pull request closed without merging**